test/integration/emitter_test.cpp

```diff
@@ -382,6 +382,20 @@ TEST_F(EmitterTest, ScalarFormat) {
       "crazy\tsymbols that we like");
 }
 
+TEST_F(EmitterTest, LiteralWithoutTrailingSpaces) {
+  out << YAML::BeginMap;
+  out << YAML::Key << "key";
+  out << YAML::Literal;
+  out << "expect that with two newlines\n\n"
+         "no spaces are emitted in the empty line";
+  out << YAML::EndMap;
+
+  ExpectEmit(
+      "key: |\n"
```

Comment:
I meant a multi line literal like this:

https://programming-idioms.org/idiom/48/multi-line-string-literal/2680/cpp

Reply:
Isn't it better to emphasize (or easier to see) that no spaces are in between the newline characters in a regular string?

Reply:
Fair enough.